truffle.js

```diff
@@ -46,7 +79,8 @@ module.exports = {
     simulated_live: {
       network_id: "*",
       host: "localhost",
-      port: 8545
+      port: 8545,
+      provider: engine // Our costume instance
```

Comment:
1. `custom` not `costume`
2. when you import nano support as above just do `provider: nano.start()` or smth similar